### ISSUE TITLE
Issue with 0.7.4 - UnhandledPromiseRejectionWarning: WebpackOptionsValidationError: Invalid configuration object

### ISSUE THREAD

**Issue by oodesign** [skpm/skpm#262]
Hi!
I'm getting this exception and can't get the plugin to build without the warning. Any clue?

(node:20011) UnhandledPromiseRejectionWarning: WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.node should be one of these:
   false | object { Buffer?, __dirname?, __filename?, console?, global?, process?, … }
   -> Include polyfills or mocks for various node stuff.
   Details:
    * configuration.node should be false
    * configuration.node should be an object.
    at webpack (/Users/ooto/Projects/sketch-styler/node_modules/webpack/lib/webpack.js:31:9)
    at /Users/ooto/Projects/sketch-styler/node_modules/@skpm/builder/lib/build.js:148:46
    at Generator.next (<anonymous>)
    at step (/Users/ooto/Projects/sketch-styler/node_modules/@skpm/builder/lib/build.js:298:191)
    at /Users/ooto/Projects/sketch-styler/node_modules/@skpm/builder/lib/build.js:298:361
(node:20011) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:20011) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by mathieudutour** [comment]
do you have a `webpack.skpm.config.js` file? if so, what's inside?

**Comment by oodesign** [comment]
Yes I do. It's the following:

module.exports = function (config) {
  config.module.rules.push({
    test: /\.(html)$/,
    use: [{
        loader: "@skpm/extract-loader",
      },
      {
        loader: "html-loader",
        options: {
          attrs: [
            'img:src',
            'link:href'
          ],
          interpolate: true,
        },
      },
    ]
  })
  config.module.rules.push({
    test: /\.(css)$/,
    use: [{
        loader: "@skpm/extract-loader",
      },
      {
        loader: "css-loader",
      },
    ]
  })
}

**Comment by mathieudutour** [comment]
hum weird. In your config, could you `console.log(config.node)` and see what's up?

**Comment by oodesign** [comment]
Never done it before... How do I do it? Within the plugin code you mean? Or from Terminal in a specific folder?

**Comment by mathieudutour** [comment]
no, just in your `webpack.skpm.config.js` file, inside the function.

Then run `npm run build` and it should print it

**Comment by oodesign** [comment]
Thanks. It gets printed twice (false first and true later):
`[1/6] 🖨  Copied src/manifest.json in 3ms
[2/6] ⛏  Copied assets/icon-sr.png
[3/6] 🔧  Copied assets/icon.png
config.node
false
config.node
true
(node:3287) UnhandledPromiseRejectionWarning: `

**Comment by mathieudutour** [comment]
ah so it's the `true` that generates the warning. Let me check where it's coming from

**Comment by mathieudutour** [comment]
ah right so I think I already fixed it, but haven't published it yet

**Comment by mathieudutour** [comment]
should be fixed in 0.7.5

**Comment by oodesign** [comment]
Wow, that was fast! 🤩

Just updated skpm builder and now everything seems OK.

> skpm-build

```
[1/6] 🖨  Copied src/manifest.json in 2ms
[2/6] 🛠  Copied assets/icon-sr.png
[3/6] ⛏  Copied assets/icon.png
config.node
false
config.node
undefined
config.node
undefined
[4/6] ⚒  Built resources/webview.js in 817ms
[5/6] 🔨  Built resources/register.js in 830ms
[6/6] 🔧  Built ./AssignStylesAutomatically.js in 1643ms
success Plugin built

```

Thanks a lot! 🥰
